src/dvclive/live.py

```diff
@@ -265,11 +265,19 @@ def _init_dvc(self):  # noqa: C901
             self._include_untracked.append(self.dir)
 
     def _init_dvc_file(self) -> str:
-        if isinstance(self._dvcyaml, str):
-            if os.path.basename(self._dvcyaml) == "dvc.yaml":
-                return self._dvcyaml
-            raise InvalidDvcyamlError
-        return "dvc.yaml"
+        if self._dvcyaml is None:
+            return "dvc.yaml"
+        if isinstance(self._dvcyaml, bool):
+            return "dvc.yaml"
```

Comment:
Are we okay with returning `dvc.yaml` in case of `dvcyaml=False`?

Reply:
But this is unrelated to this PR. We were already doing that.

Reply:
@skshetry as pointed out by @dberenbaum, "the slight justification for still setting it to dvc.yaml is that if someone has set Live(dvcyaml=False) but then manually calls make_dvcyaml(), it will still write to dvc.yaml. "dvc.yaml" is being returned ".